Formula/infer.rb

```diff
@@ -15,12 +15,21 @@ class Infer < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "camlp4" => :build
   depends_on "cmake" => :build
+  depends_on "gnu-sed" => :build
```

Comment:
Is this really needed? System sed won't work?

Reply:
I added `gnu-sed` as a build dep since when I build infer locally I noticed the configure script checking for gnu sed. I'll remove this and see if system sed works.

Reply:
Yes please. As a rule of thumb, don't just add things with mac counterparts because a config script checks for them. The goal is to find the right balance between widespread demand and functionality with the minimal possible set of dependencies. Whenever macOS provides something like sed, always assume that it will work (and test it) before adding an additional dependency.

Reply:
that makes sense. first time writing a larger brew formula, so I appreciate the advice.